tfe/testing.go

```diff
@@ -50,6 +61,12 @@ func skipIfEnterprise(t *testing.T) {
 	}
 }
 
+func skipUnlessRunTasksDefined(t *testing.T) {
+	if _, ok := os.LookupEnv(RunTasksURLEnvName); !ok {
```

Comment:
`LookupEnv` will return `ok == true` even if `RUN_TASKS_URL=""`, so we should ensure the string is not empty before we write off the url as defined.

Reply:
Done.